bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/messages.properties

```diff
@@ -1150,3 +1150,9 @@ OpenDelayedUrlAction_title=Open URL
 editorAssociationOverride_error_couldNotCreate_message=The ''{0}'' extension from plug-in ''{1}'' to the ''org.eclipse.ui.ide.editorAssociationOverride'' extension point failed to load the editor association override class.
 editorAssociationOverride_error_invalidElementName_message=An extension from plug-in ''{0}'' to the ''org.eclipse.ui.ide.editorAssociationOverride'' extension point was ignored because it contains the following invalid element: ''{1}''.
 editorAssociationOverride_error_invalidExtension_message=The ''{0}'' extension from plug-in ''{1}'' to the ''org.eclipse.ui.ide.editorAssociationOverride'' extension point will be ignored because it contains invalid attributes.
+
+User=User: {0}\n
```

Comment:
```suggestion
Workspace_Lock_Owner_User=User: {0}\n
```

same for the other constants. I would also check if it is maybe better to have one label for the "key" and one for the value, then no placeholder is required and one can align the text better.